sdk/search/search/review/search.api.md

```diff
@@ -306,6 +301,21 @@ export interface IndexDocuments extends OperationOptions {
     throwOnAnyFailure?: boolean;
 }
 
+// @public
+export type IndexDocumentsAction<T> = {
+    __actionType: IndexActionType;
```

Comment:
not super pumped about the double-underscore. Had a convo offline on this, considered space prefix (with autorest doesn't like), or an envelope (which seemed overkill for this case). Given the nichety of this API, I'll allow it.

Reply:
😅